Eatery/Onboarding/Controllers/OnboardingLoginViewController.swift

```diff
@@ -55,18 +56,35 @@ class OnboardingLoginViewController: OnboardingViewController {
         setUpErrorView()
         setUpNetidViews()
         setUpPasswordViews()
+        setUpButton()
         setUpSkipButton()
+
+        contentView.snp.makeConstraints { make in
+            make.width.equalToSuperview()
+            make.height.equalTo(stackView)
+        }
+
+        NotificationCenter.default.addObserver(self, selector: #selector(self.keyboardWillShow), name: NSNotification.Name.UIKeyboardWillShow, object: nil)
```

Comment:
Can use type inference to shorten this line to `NotificationCenter.default.addObserver(self, selector: #selector(keyboardWillShow), name: .UIKeyboardWillShow, object: nil)`

It might not be possible to shorten to `.UIKeyboardWillShow` since Eatery uses an older version of Swift, not sure